### ISSUE TITLE
The My Workspace actions works incorrectly

### ISSUE THREAD

**Issue by RomanNikitenko** [eclipse-che/che#13956]
### Describe the bug
The bug is relevant to master branch assembly:
- theia version: https://github.com/theia-ide/theia/commit/3587c23
- che-theia version: https://github.com/eclipse/che-theia/commit/582b1c5

The `My Workspace` panel actions works incorrectly  
Please see the video: https://youtu.be/14PZ3ARjZBc

### Che version

 - [ ] latest
 - [ ] nightly
 - [x] other: 7.0.0-rc-4.0-SNAPSHOT

### Steps to reproduce
1. To use `eclipse/che-theia:next` image define in devfile:
```
- 
    alias: theia-editor
    reference: >-
      https://raw.githubusercontent.com/eclipse/che-plugin-registry/master/v3/plugins/eclipse/che-theia/next/meta.yaml
    type: cheEditor

```
2. `chectl workspace:start -f devfile.yaml`
3. Try to use `My Workspace` panel actions, for example `New Terminal` action or execute a command.
4. You can see that:
-  terminal widget is opened but you can not type any command.
- errors in the browser console
- quick open widget open instead of running the selected command for execution from `My Workspace` panel.

### Expected behavior
-  terminal widget is opened and ready for use.
- the selected command is running.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`) 
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (minikube v1.0.0 on linux (amd64))
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl chectl/0.0.2-6d3894f linux-x64 node-v10.4.1)
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer 
       - [ ] Windows
       - [x] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify) 
   - [ ] other: please specify

**Comment by l0rd** [comment]
@RomanNikitenko that's related to theia:next, not Che 7 right?

**Comment by RomanNikitenko** [comment]
yes, the issue is related to che-theia:next

**Comment by l0rd** [comment]
Ok thanks. Set milestone to 7.1

**Comment by RomanNikitenko** [comment]
I investigated the issue and described the cause of the problem [here](https://github.com/eclipse/che/issues/14204#issuecomment-520590519).
The bug was relevant to `che-theia:next` only, but recently the corresponding changes was merged to `che-7.0.0` branch as well, so now it's relevant to `7.0.0` too.

**Comment by l0rd** [comment]
> I investigated the issue and described the cause of the problem [here](https://github.com/eclipse/che/issues/14204#issuecomment-520590519).
> The bug was relevant to `che-theia:next` only, but recently the corresponding changes was merged to `che-7.0.0` branch as well, so now it's relevant to `7.0.0` too.

How is that possible? @evidolob aren't we blocking upstream PRs to get to 7.0.0 branch?

**Comment by gorkem** [comment]
This is a fix it for 7.0 issue.

**Comment by vinokurig** [comment]
fixed in https://github.com/theia-ide/theia/pull/5961
